### PR TITLE
Review documentation (issue #97)

### PR DESCRIPTION
**Pull request by carsten-kuebler**
This PR addresses issue #97.

Review doxygen links.
Review documentation layout in proto (80 chars).
Review doxgen commands.
Remove typos
**Rename DetectionHeader --> SensorDetectionHeader**

Review TrafficSignTyps and **add some traffic signs.**
Add references to traffic sign images.
**Generalize traffic signs** (30km/h zone --> speed limit zone & value & value_unit)

Rename **repeated field names (plural or list as suffix).**
Add units to message field documentation.

### CODE REVIEW

**Comment by carsten-kuebler** [comment]
@CarloVanDriestenBMW See all "important" changes in bold.

File: osi_common.proto [hunk 1]
```diff
@@ -191,13 +194,13 @@ message MountingPosition
 // 
 // Azimuth and elevation are defined as the rotations that would have to be 
 // applied to the local frame (e.g sensor frame definition in 
-// \c OSI:DetectionHeader) to make its x-axis point towards the referenced point
+// \c DetectionHeader) to make its x-axis point towards the referenced point
```
Comment (CarloVanDriestenBMW) [inline osi_common.proto]: SensorDetectionHeader?

Comment (carsten-kuebler) [reply osi_common.proto]: @CarloVanDriestenBMW I changed all DetectionHeader to SensorDetecionHeader.

Comment (carsten-kuebler) [reply osi_common.proto]: Thx

**Pull request merged**